app/views/participant_signup/_family_history.html.erb

```diff
@@ -0,0 +1,27 @@
+<%= f.inputs :name => "Family History Information" do -%>
+
+    <%= f.input :first_generation, :as => :select, :label => "Did your parents graduate from college?", :collection => { "No (I am first generation to college)" => true, "Yes (I am NOT first generation to college)" => false } %>
+	<%= f.input :family_members_who_went_to_college, :input_html => { :size => "90%" } %>
+	<%= f.input :family_members_graduated, :label => "Did any of them graduate?" %>    
+	
+	<%= f.inputs :name => "Attended school outside USA", :class => "inline" do -%>
+		<%= f.input :attended_school_outside_usa, :label => "Yes" %>
+		<%= f.input :countries_attended_school_outside_usa, :label => "In what countries?" %>
+	<% end %>
+	
+	<%= f.inputs :name => "Grades attended outside USA", :class => "inline" do -%>
+		<%- for grade in 1..12 -%>
+			<%= f.input "attended_grade_#{grade}_outside_usa", :label => grade.to_s %>
+		<% end %>
+	<% end %>
+
+	<%= f.inputs :name => "Marriage/Children", :class => "inline" do -%>
+		<%= f.input :married %>
+		<%= f.input :aliases, :label => "Maiden Name", :as => :string %>
+		<%= f.input :number_of_children, :collection => 0..5, :as => :select %>
+	<% end %>
+	
+	<%= f.input :subsidized_housing, :label => "My family receives public assistance or lives in federally subsidized housing", :as => :radio %>
+	<%= f.input :homeless, :label => "I am currently experiencing unstable housing (live with family members other than my parents/legal guardians, couch surfing, live with friends, homeless, live in a motel or hotel)" %>
```

Comment:
@mattharris5 There are some labels which are changed, like for homeless and subsidized housing, and many other fields. Will using the Help text model also account for such cases?